src/server/error.ts

```diff
@@ -33,3 +33,9 @@ export class MissingOperationError extends OperationError {
         super('MISSING', operation, `Missing operation on index ${index}`);
     }
 }
+
+export class DriveAlreadyExistsError extends Error {
+    constructor() {
```

Comment:
Let's also store the drive id on the error.

Reply:
sure! added here: [e1097a7](https://github.com/powerhouse-inc/document-drive/pull/277/commits/e1097a75a7b8c4475310127217252cc2a3e8cae5)